src/MigrationDummyAnalyzer.php

```diff
@@ -0,0 +1,27 @@
+<?php declare(strict_types = 1);
+
+namespace ShipMonk\Doctrine\Migration;
+
+class MigrationDummyAnalyzer implements MigrationsAnalyzer
```

Comment:
I'd call it `MigrationDefaultAnalyzer`, similar to `MigrationDefaultVersionProvider` and `MigrationDefaultExecutor`

Reply:
i understand, but the difference is that DefaultExecutor and DefaultVP actually do something

Reply:
The default one does too: "use before stage for all sqls".

Reply:
ok then : ]